src/jaxsim/mujoco/loaders.py

```diff
@@ -598,21 +611,106 @@ def convert(
+        distance: float | int | npt.NDArray = 3,
+        azimut: float | int | npt.NDArray = 90,
+        elevation: float | int | npt.NDArray = -45,
+        degrees: bool = True,
+        **kwargs,
+    ) -> MujocoCamera:
+        """
+        Create a custom camera that looks at a target point.
 
-        if len(kwargs["xyaxes"].split()) != 6:
-            raise ValueError("xyaxes must have six values separated by space")
+        Note:
+            The choice of the parameters is easier if we imagine to consider a target
+            frame `T` whose origin is located over the lookat point and having the same
+            orientation of the world frame `W`. We also introduce a camera frame `C`
+            whose origin is located over the lower-left corner of the image, and having
+            the x-axis pointing right and the y-axis pointing up in image coordinates.
+            The camera renders what it sees in the -z direction of frame `C`.
 
-        return cls(**kwargs)
+        Args:
+            camera_name: The name of the camera.
+            lookat: The target point to look at (origin of `T`).
+            distance:
+                The distance from the target point (displacement between the origins
+                of `T` and `C`).
+            azimut:
+                The rotation around z of the camera. With an angle of 0, the camera
+                would loot at the target point towards the positive x-axis of `T`.
+            elevation:
+                The rotation around the x-axis of the camera frame `C`. Note that if
+                you want to lift the view angle, the elevation is negative.
+            degrees: Whether the angles are in degrees or radians.
+            **kwargs: Additional camera parameters.
+
+        Returns:
+            The custom camera.
+        """
+
+        # Start from a frame whose origin is located over the lookat point.
+        # We initialize a -90 degrees rotation around the z-axis because due to
+        # the default camera coordinate system (x pointing right, y pointing up).
+        W_H_C = np.eye(4)
+        W_H_C[0:3, 3] = np.array(lookat)
+        W_H_C[0:3, 0:3] = Rotation.from_euler(
+            seq="Z", angles=-90, degrees=degrees
+        ).as_matrix()
+
+        # Process the azimut.
+        R_az = Rotation.from_euler(seq="Z", angles=azimut, degrees=degrees).as_matrix()
+        W_H_C[0:3, 0:3] = W_H_C[0:3, 0:3] @ R_az
+
+        # Process elevation.
+        R_el = Rotation.from_euler(
+            seq="X", angles=-elevation, degrees=degrees
+        ).as_matrix()
+        W_H_C[0:3, 0:3] = W_H_C[0:3, 0:3] @ R_el
+
+        # Process distance.
+        tf_distance = np.eye(4)
+        tf_distance[2, 3] = distance
+        W_H_C = W_H_C @ tf_distance
+
+        # Extract the position and the quaternion.
+        p = W_H_C[0:3, 3]
+        Q = Rotation.from_matrix(W_H_C[0:3, 0:3]).as_quat()[np.array([3, 0, 1, 2])]
```

Comment:
```suggestion
        Q = Rotation.from_matrix(W_H_C[0:3, 0:3]).as_quat(scalar_first=True)
```

Reply:
Right, the `Rotation` from plain SciPy has this kind-of new option. I also need to add `scipy` as `viz` dependency.

Reply:
Thanks! Do you think it would make sense to just use `jaxlie` or `jaxsim.math.Rotation` to avoid adding an additional dependency? Otherwise we should also push a new build to conda-forge

Reply:
For the time being, I'd stick with plain `scipy`. If you notice, the `jaxsim.mujoco` package is standalone, it does not depend on any jax-based library. This was done on purpose if, at some point, we want to port it outside JaxSim. Scipy is not a lightweight dependency, but it will be likely already installed in the majority of user's setups.

> Otherwise we should also push a new build to conda-forge

This needs to be done not before the next tagged release right?

Reply:
> For the time being, I'd stick with plain scipy

Alright!


> This needs to be done not before the next tagged release right?

Yes, we can publish a new build without bumping the version

Reply:
> > This needs to be done not before the next tagged release right?
> 
> Yes, we can publish a new build without bumping the version

Yep but this was not required by previous versions if I don't mistake. It's something new here.

Reply:
Done in https://github.com/conda-forge/jaxsim-feedstock/pull/6